### ISSUE TITLE
Network Protocol Documentation?

### ISSUE THREAD

**Issue by astronouth7303** [Dees-Troy/SimpleCoil#3]
What's the protocol used by the TCP and UDP servers?

**Comment by Dees-Troy** [comment]
TCP and UDP are protocols. I presume that you're more looking for documentation of how the devices communicate with each other. Unfortunately I did not document this aspect of SimpleCoil beyond a few code comments. You'll either have to review the code itself or add or uncomment some logging such as these:
https://github.com/Dees-Troy/SimpleCoil/blob/master/app/src/main/java/com/simplecoil/simplecoil/UDPListenerService.java#L94
https://github.com/Dees-Troy/SimpleCoil/blob/master/app/src/main/java/com/simplecoil/simplecoil/UDPListenerService.java#L501

If you have questions after digging through the code, feel free to ask. The network protocol only matters if you are trying to make your own app that's compatible with SimpleCoil. Otherwise, you can roll your own network stack.

**Comment by joshkrebs** [comment]
May I just say how amazing this is?!?! Your SimpleCoil app has fantastic customization features and the idea to leverage an existing network outside the Recoil hub is brilliant! The indoor arena size and play options explode with possibilities. You are AWESOME and you have given me the ability to create some student event options built around recoil. You see, I teach at an all-computer-science college in Utah and this is the kind of thing we LOVE here. I can see a day where my students make it their junior/senior project to bootstrap their own app from SimpleCoil and/or add new features to the base app.

TL;DR - You are amazing, this app is amazing, and I can't wait to get my students hooked on your app and the Recoil system.
